diagnostic.md

```diff
@@ -94,4 +96,6 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+'Exercise'. 'jason', 'weeks'. [1,2].
+
+Definitely not feeling confident on this one^
```

Comment:
You got the values of teamMember['hobby'] and teamMember['name']['surname'] correct.  The value of teamMember['name'] is the nested object { given: 'Jason', surname: 'Weeks' }.  Your answer would actually be teamMember['name']['given'].